### DISCUSSION TITLE
Welcome to our community 🎉

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#1] [Announcements]
We decided to create this space as we often receive great ideas, feedback or feature requests from you - our users, but so far we had no way of exposing this information to the wider community or allowing you to discuss it with fellow users (or our team).
By launching this repository we hope to bridge the gap, allowing you to participate in shaping the direction and quality of the product while giving everyone greater visibility into the feedback we receive. 

You will see two Categories available: 

- Announcements - our team will be posting here about any updates, releases, and other news. 
- Ideas-and-feature-requests - this is where we welcome all product-related contributions! If you have a product idea or if there is a feature that you would like to see please share it here. You can also ⬆️ upvote ideas posted by others or interact in the comments. 

We are excited to kick off this journey with you! 

Disclaimer: _Although we can't wait to hear your thoughts and you can expect engagement from us, please note that this is not a support channel. For bugs and issues, we strongly encourage you to contact support@gitbook.com as we will not be monitoring this space 24/7._

**Comment by volkantash** [comment]
Hi

You can add this discussion tab to [Your github organization tab](https://github.com/GitbookIO) 

Check it: https://github.blog/changelog/2022-04-12-organization-discussions/

![resim](https://user-images.githubusercontent.com/25375593/215831468-ed56e09d-d62d-4462-97ae-131aa20bcdd5.png)
